services/core/shared/core/compat/sdk_v5/peers.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const coreApi = require('./coreApi');
+
+const peerStates = {
+	CONNECTED: 'connected',
+	DISCONNECTED: 'disconnected',
+};
+
+const getPeers = async () => {
+	const connectedPeers = await coreApi.getPeers(peerStates.CONNECTED);
+	connectedPeers.data.forEach(peer => {
+		peer.state = peerStates.CONNECTED;
+		peer.height = peer.options.height;
+		peer.ip = peer.ipAddress;
+		return peer;
+	});
+
+	const disconnectedPeers = await coreApi.getPeers(peerStates.DISCONNECTED);
+	disconnectedPeers.data.forEach(peer => peer.state = peerStates.DISCONNECTED);
```

Comment:
are we not getting `height` and `ip` for the disconnected peers as other versions?